ffi/framebuffer_mxcfb.lua

```diff
@@ -502,6 +514,17 @@ function framebuffer:init()
         self.waveform_flashui = self.waveform_ui
         self.waveform_full = C.WAVEFORM_MODE_GC16
         self.waveform_partial = C.WAVEFORM_MODE_GC16
+    elseif self.device:isPRST() then
+        require("ffi/mxcfb_sony_h")
+
+        self.mech_refresh = refresh_prst
+        self.mech_wait_update_complete = prst_mxc_wait_for_update_complete
+
+        self.waveform_fast = C.WAVEFORM_MODE_DU
```

Comment:
Unless there's a platform-specific issue with A2 (like there is on REAGL), we usually go with A2 instead of DU for that ;).